Remotion/ObjectBinding/Web/UI/Controls/Validation/Factories/ValidationBusinessObjectBoundEditableWebControlValidatorFactory.cs

```diff
@@ -112,63 +112,60 @@ public IEnumerable<BaseValidator> CreateValidators (IBocTextValue control, bool
     {
       ArgumentUtility.CheckNotNull("control", control);
 
```

Comment:
We got a test fixture for this class. it should fail for readonly now. please check and update the tests.